README.md

```diff
@@ -6,60 +6,65 @@
-trust bundles in Envoy. Please see the [SPIRE Agent configuration guide](/doc/spire_agent.md#agent-configuration-file) for more information.
+- See [CONTRIBUTING](https://github.com/spiffe/spire/blob/master/CONTRIBUTING.md) to get started.
+- Use [GitHub Issues](https://github.com/spiffe/spire/issues) to request features or file bugs.
+- See [GOVERNANCE](https://github.com/spiffe/spiffe/blob/master/GOVERNANCE.md) for SPIFFE and SPIRE governance policies.
 
-# Upgrading SPIRE
+## Further Reading
 
-SPIRE Server supports zero-downtime upgrades when there's more than one SPIRE Server in the cluster. Please see the [Managing Upgrades/Downgrades](doc/upgrading.md) guide for more information on SPIRE version compatibility and supported upgrade paths.
+- The [Scaling SPIRE guide](/doc/scaling_spire.md) covers design guidelines, recommendations, and deployment models.
+- For an explanation of how SPIRE compares to related systems such as secret stores, identity providers, authorization policy engines and service meshes see [comparisons](https://spiffe.io/spire/comparisons/).
 
-# Getting Help
+## Security
 
-If you have any questions about how SPIRE works, or how to get it up and running, the best place to ask questions is the [SPIFFE Slack Organization](https://slack.spiffe.io/). Most of the maintainers monitor the #spire channel there, and can help direct you to other channels if need be. Please feel free to drop by any time!
+### Security Assessments
 
-# Community
+The [CNCF Special Interest Group for Security]([https://github.com/cncf/sig-security](https://github.com/cncf/sig-security)) has conducted two assessments on SPIFFE and SPIRE design and configuration with respect to security.  The following links contain summary reports as well as the threat modeling material produced as part of the asssessment processes:
 
-The SPIFFE community, and [Scytale](https://scytale.io) in particular, maintain the SPIRE project.
-Information on the various SIGs and relevant standards can be found in
-https://github.com/spiffe/spiffe.
+- [Scrutinizing SPIRE to Sensibly Strengthen SPIFFE Security](https://blog.scytale.io/scrutinizing-spire-security-9c82ba542019)
+- [SIG Security SPIFFE/SPIRE Security Assessment Summary](https://github.com/cncf/sig-security/tree/master/assessments/projects/spiffe-spire)
+- [SIG Security SPIFFE/SPIRE Security Assessment Document](https://github.com/cncf/sig-security/blob/master/assessments/projects/spiffe-spire/self-assessment.md)
 
-The SPIFFE and SPIRE governance policies are detailed in
-[GOVERNANCE](https://github.com/spiffe/spiffe/blob/master/GOVERNANCE.md).
+### Reporting Security Vulnerabilities
 
+If you've found a vulnerability or a potential vulnerability in SPIRE please let us know at security@spiffe.io. We'll send a confirmation email to acknowledge your report, and we'll send an additional email when we've identified the issue positively or negatively.
```

Comment:
Maybe this can be reworded and expanded in a way that we express how important and instrumental is security in the SPIRE project, and we encourage to report security issues.
Aligned with that, it would be good if we (at some point) provide a PGP public key for the communication with us.

Reply:
Agreed. 

Ideally this can submitted on the web using HTTPS (TLS) or an email encrypted using OpenPGP.

In order to maintain the projects CII best practices badge, it must be ensured that project's initial response time for any vulnerability report received is less than or equal to 14 days.